cypress/integration/marc-authority/marc-authority-delete.spec.js

```diff
@@ -0,0 +1,75 @@
+import getRandomPostfix from '../../support/utils/stringTools';
+import { getLongDelay } from '../../support/utils/cypressTools';
+import TestTypes from '../../support/dictionary/testTypes';
+import Features from '../../support/dictionary/features';
+import DevTeams from '../../support/dictionary/devTeams';
+import Permissions from '../../support/dictionary/permissions';
+import TopMenu from '../../support/fragments/topMenu';
+import DataImport from '../../support/fragments/data_import/dataImport';
+import MarcAuthority from '../../support/fragments/marcAuthority/marcAuthority';
+import JobProfiles from '../../support/fragments/data_import/job_profiles/jobProfiles';
+import MarcAuthoritiesSearch from '../../support/fragments/marcAuthority/marcAuthoritiesSearch';
+import MarcAuthorities from '../../support/fragments/marcAuthority/marcAuthorities';
+import Users from '../../support/fragments/users/users';
+import MarcAuthoritiesDelete from '../../support/fragments/marcAuthority/marcAuthoritiesDelete';
+
+describe('MARC Authority management', () => {
+  const testData = {
+    userProperties: { name:'testname' },
+    uniqueFileName: `autotestFile.${getRandomPostfix()}.mrc`
+  };
+  const importFile = (profileName) => {
```

Comment:
this func seems usable. move it to related fragment please

Reply:
done